networks/parallel.ts

```diff
@@ -0,0 +1,34 @@
+import { Config } from './types'
+
+export default {
+  polkadot: {
+    name: 'parallel' as const,
+    endpoint: 'wss://rpc.parallel.fi',
+  },
+  kusama: {
+    name: 'heiko' as const,
+    endpoint: 'wss://parallel-heiko.api.onfinality.io/public-ws',
+  },
+  config: ({ alice }) => ({
+    storages: {
+      System: {
+        account: [[[alice.address], { providers: 1, data: { free: 1000 * 1e12 } }]],
+      },
+      Assets: {
+        account: [
+          [[heiko.ausd, alice.address], { balance: 100 * 1e12 }], // AUSD
```

Comment:
the value is different for each networks so should be something like this https://github.com/AcalaNetwork/e2e-tests/blob/2842cef66c3ca289fe69c029b8088e6b821c101b/networks/acala.ts#L13